tensorflow/core/public/version.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 #define TF_MAJOR_VERSION 0
 #define TF_MINOR_VERSION 12
-#define TF_PATCH_VERSION head
+#define TF_PATCH_VERSION 0-rc0
```

Comment:
Wait, I missed this.
I thought the agreement was "0.head"
0 as major version, "head" as patch version?